wrappers/nodejs/test/test-functional-online.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2017 Intel Corporation. All rights reserved.
```

Comment:
nit, 2018

Reply:
The eslint-plugin-header was using hard-coded header.js file to compare the header content, I will change it to use regular expressions. Otherwise it would fail during the lint check.